pyaedt/filtersolutions_core/lumped_termination_impedance.py

```diff
@@ -0,0 +1,391 @@
+            c_char_p,
+            c_bool,
+        ]
+        self._dll.updateComplexTableRow.restype = c_int
+
+        self._dll.appendComplexTableRow.argtypes = [
+            c_char_p,
+            c_char_p,
+            c_char_p,
+            c_bool,
+        ]
+        self._dll.appendComplexTableRow.restype = c_int
+
+        self._dll.insertComplexTableRow.argtypes = [
+            c_int,
+            c_char_p,
+            c_char_p,
+            c_char_p,
+            c_bool,
+        ]
+        self._dll.insertComplexTableRow.restype = c_int
+
+        self._dll.removeComplexTableRow.argtypes = [c_int, c_bool]
+        self._dll.removeComplexTableRow.restype = c_int
+
+        self._dll.setLumpedComplexDefinition.argtypes = [c_char_p, c_bool]
+        self._dll.setLumpedComplexDefinition.restype = c_int
+        self._dll.getLumpedComplexDefinition.argtypes = [c_char_p, c_bool, c_int]
+        self._dll.getLumpedComplexDefinition.restype = c_int
+
+        self._dll.setLumpedComplexReactanceType.argtypes = [c_char_p, c_bool]
+        self._dll.setLumpedComplexReactanceType.restype = c_int
+        self._dll.getLumpedComplexReactanceType.argtypes = [c_char_p, c_bool, c_int]
+        self._dll.getLumpedComplexReactanceType.restype = c_int
+
+        self._dll.setLumpedComplexImpCompensateEnabled.argtypes = [c_bool, c_bool]
+        self._dll.setLumpedComplexImpCompensateEnabled.restype = c_int
+        self._dll.getLumpedComplexImpCompensateEnabled.argtypes = [
+            POINTER(c_bool),
+            c_bool,
+        ]
+        self._dll.getLumpedComplexImpCompensateEnabled.restype = c_int
+
+        self._dll.setLumpedComplexCompOrder.argtypes = [c_int, c_bool]
+        self._dll.setLumpedComplexCompOrder.restype = c_int
+        self._dll.getLumpedComplexCompOrder.argtypes = [POINTER(c_int), c_bool]
+        self._dll.getLumpedComplexCompOrder.restype = c_int
+
+    def table_type_to_bool(self):
+        """Set a flag to recognize source and load complex tables.
+
+        Returns
+        -------
+        bool
+        """
+        if self.table_type.value == TerminationType.SOURCE.value:
+            return False
+        else:
+            return True
+
+    @property
+    def row_count(self) -> int:
+        """The count of accumulated complex impedances in the complex impedances tables.
+        The default is ``3``.
+
+        Returns
+        -------
+        int
+        """
+        table_row_count = c_int()
+        status = self._dll.getComplexTableRowCount(byref(table_row_count), self.table_type_to_bool())
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return int(table_row_count.value)
+
+    def row(self, row_index):
+        """Return frequencies and complex impedances at given index of the complex impedances tables.
+
+        Parameters
+        ----------
+        row_index: int
+            The row index on complex impedances tables. Starting value is 0 and maximum value is 99.
```

Comment:
```suggestion
        """Get frequencies and complex impedances at a given index of the complex impedance tables.

        Parameters
        ----------
        row_index: int
            Row index on complex impedance tables. The starting value is ``0``, and the maximum value is ``99``.
```
Here I decided that using complex impedance tables is likely the preferred route. Working with singular and plural possessive forms of a word can be tricky. The same concern I noted earlier exists for the following "Returns" section.